commands/upload.ts

```diff
@@ -317,11 +336,24 @@ exports.builder = yargs => {
     default: false,
   });
 
-  addConfigOptions(yargs);
-  addAccountOptions(yargs);
-  addCmsPublishModeOptions(yargs, { write: true });
```

Comment:
Looks like we might have dropped addCMSPublishModeOptions. It can go in the builder